plugins/eni/commands/commands.go

```diff
@@ -78,6 +81,17 @@ func add(args *skel.CmdArgs, engine engine.Engine) error {
 	}
 	log.Debugf("Found ipv4 gateway and netmask for ENI: %s %s", ipv4Gateway, ipv4Netmask)
 
+	_, ipv4Net, err := net.ParseCIDR(fmt.Sprintf("%s/%s", ipv4Gateway, ipv4Netmask))
+	if err != nil {
+		return errors.Wrapf(err, "add eni: parse cidr failed: %s", fmt.Sprintf("%s/%s", ipv4Gateway, ipv4Netmask))
```

Comment:
Can you please add more context to this error message around where it's generated (or make it more customer friendly)?

`"add eni: failed to parse gateway netmask: %s"`